qtpy/QtCore.py

```diff
@@ -7,7 +7,7 @@
 # -----------------------------------------------------------------------------
 
 """Provides QtCore classes and functions."""
-
+from typing import TYPE_CHECKING
 from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, QtBindingsNotFoundError
```

Comment:
```suggestion
from typing import TYPE_CHECKING

from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, QtBindingsNotFoundError
```

Should be in a separate import group (stdlib vs third party), per PEP 8 and standard convention.

Reply:
sure!  have you considered pre-commit here?